compiler/vmops.nim

```diff
@@ -22,8 +22,9 @@ when declared(math.signbit):
   # ditto
   from std/math as math3 import signbit
 
+
 from std/envvars import getEnv, existsEnv, delEnv, putEnv, envPairs
-from std/os import dirExists, fileExists, walkDir, getAppFilename
+from std/os import walkDir, getAppFilename, dirExists, fileExists
```

Comment:
It will be changed to `from std/private/oscommon import dirExists, fileExists` in the following PRs because of bootstrapping.